.cirrus.yml

```diff
@@ -73,105 +67,32 @@ linux_task:
       PY_VER: "3.7"
     env:
       PY_VER: "3.8"
-      COVERAGE: "pytest-cov codecov"
-  name: "Linux: py${PY_VER}"
+      COVERAGE: "true"
+  name: "${CIRRUS_OS}: py${PY_VER} tests"
   container:
     image: gcc:latest
   env:
     PATH: ${HOME}/miniconda/bin:${PATH}
-    CODECOV_TOKEN: "ENCRYPTED\
-      [1ed538b97a8d005bdd5ab729de009ac38a2b53389edb0912\
-      d2e76f5ce1e71c5f7bdea80a79492b57af54691c8936bdc7]"
```

Comment:
The `CODECOV_TOKEN` is now configured via the `cirrus-ci` `SciTools-incubator/iris-esmf-regrid` project configurations accessible from the browser